client/state/sites/features/reducer.js

```diff
@@ -25,6 +29,26 @@ function updateSiteState( state, siteId, attributes ) {
 	} );
 }
 
+/**
+ * Given an object of features keyed by siteId, updates all features at once.
+ *
+ * @param {object} state The current features state
+ * @param {object} features An object containing feature arrays keyed by siteId
+ * @returns The new state
+ */
+function updateBulkFeatures( state, features ) {
+	const newState = {};
+	for ( const siteId in features ) {
+		newState[ siteId ] = {
+			error: null,
+			hasLoadedFromServer: true,
+			isRequesting: false,
+			data: features,
+		};
```

Comment:
This looks duplicated with the rest of the site features reducer. Can we create a helper function and use it for this part of the reducer, in order to avoid repetition and the possibility of errors due to missing keys or misshapen format?

Reply:
I decided to use the `initialSiteState` and then only set the properties which would be changed by this operation.